custom_components/tuya_local/devices/hyd_weair_plus_fan.yaml

```diff
@@ -3,93 +3,95 @@ products:
   - id: 6p7jm8rrqimnlusw
     name: D-68
 primary_entity:
-  entity: fan
+  entity: climate
   dps:
     - id: 1
       type: boolean
-      name: switch
+      name: mode
```

Comment:
For the "toggle" service call to work correctly, I think this one needs to be hvac_mode, and 103 the hidden mode.

I think the logic is easier that way too, as the current logic is probably wrong for the case dps: {"1": false, "103": true}

```
  - id: 1
    type: boolean
    name: hvac_mode
    mapping:
      - dps_val: false
        value: "off"
      - dps_val: true
        constraint: mode
        conditions:
           - dps_val: true
             value: heat
           - dps_val: false
             value: fan_only
  - id: 103
    name: mode
    type: boolean
    hidden: true
```